### PR TITLE
Add unittest for None loss weight

### PR DESCRIPTION
**Pull request by ZhiYuan-REN**
This PR is used to test default weight 'None' in loss_classification and loss_regression unitly.

### CODE REVIEW

File: tests/test_loss/test_regression_losses.py [hunk 1]
```diff
@@ -76,6 +100,17 @@ def test_mse_regression_loss():
     fake_label = torch.zeros((1, 3, 3))
     assert torch.allclose(loss(fake_pred, fake_label, None), torch.tensor(1.))
 
+    # w/o with default target weight None
```
Comment (ly015) [inline tests/test_loss/test_regression_losses.py:103]: the comment is strange here.

**Review by ly015 — APPROVED** [review]

**Comment by OpenMMLab-Coodinator** [comment]
Hi @ZhiYuan-REN ！First of all, we want to express our gratitude for your significant PR in the MMPose project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

If you are Chinese or have WeChat，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤
